workspace/notebook/py_unit_tests_functions.json

```diff
@@ -315,6 +315,128 @@
 					"    display(missing_rows_in_hrm)"
 				],
 				"execution_count": 19
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"## Define the harmonise table that only includes horizon data, and the latest record per primary id.\r\n",
+					"def harmonise_df_hzn(hrm_table: str, primary_key: str) -> DataFrame:\r\n",
+					"    df_harmonise_only_hzn: DataFrame = spark.sql(f\"\"\"\r\n",
+					"    select *\r\n",
+					"    from {hrm_db_name}.{hrm_table_final} tbl_1\r\n",
```

Comment:
in this function, the parameter `hrm_table` is not being used anywhere. And the two variables `hrm_db_name`, `hrm_table_final` that are being used don't exist. May be we can use the hrm_table instead?